docs/connect.html

```diff
@@ -95,9 +95,10 @@
 
     <p>The <code>connector.class</code> config supports several formats: the full name or alias of the class for this connector. If the connector is org.apache.kafka.connect.file.FileStreamSinkConnector, you can either specify this full name or use FileStreamSink or FileStreamSinkConnector to make the configuration a bit shorter.</p>
 
-    <p>Sink connectors also have one additional option to control their input:</p>
+    <p>Sink connectors also have a few additional options to control their input. Each sink connector must set one of the following:</p>
     <ul>
-        <li><code>topics</code> - A list of topics to use as input for this connector</li>
+        <li><code>topics</code> - A comma-separated list of topics to use as input for this connector</li>
+        <li><code>topics.regex</code> - A regular expression (compiled to <code>java.util.regex.Pattern</code>) of topics to use as input for this connector</li>
```

Comment:
Minor: the configuration property is just a Java regular expression that _will_ be compiled into a `java.util.regex.Pattern`. I'm not sure it's necessary to denote the implementation, but instead would be sufficient to just say "A Java regular expression of topics to use as input for this connector".